frontend/src/router/index.ts

```diff
@@ -771,6 +766,37 @@ const routes: Array<RouteRecordRaw> = [
       },
     ],
   },
+  {
```

Comment:
This file exceeds more than 1200 lines. Would it be better to separate routes in different files? Let the module itself manage its own routes.

Reply:
I think this file is a type of configuration file which is not often changed. So it's ok for me that in one file. wdyt? @LiuJi-Jim @ecmadao

Reply:
I personally prefer a single place to manage as it improves readability to see all routes in a single place.

e.g. a reasonable route name depends on whether it fits with the rest of the other routes.

Reply:
@tianzhou I'm with you on "reasonable route name should fit with the rest of the other routes". Yet, mostly the other routes refer to the routes belonging to the same module. 

Name a folder "routes", divide routes into separate files and import them to routes/index.js could be a better choice. 

To improve readability and avoid "path" naming conflicts, we could declare path constants as symbols(same as declaring Redux action type as symbol).

As the project pumps up, more and more routes will be configured. Putting them in one file would be hard to distinct which functionality they belong to.

Reply:
Point taking, but having a single file to record all routes is an intentional design choice. Everyone knows where to find the route config, and view its history. It's true that we have over 1000 LOC, however, most lines are identical and they are more like **config code**,  so it doesn't bear the same complexity as normal code.

Similarly, we have a single file to define API ACL routes: https://github.com/bytebase/bytebase/blob/main/server/acl_casbin_policy_dba.csv